package.json

```diff
@@ -55,6 +55,7 @@
     "react-native-slider": "0.11.0",
     "react-native-status-bar-size": "0.3.3",
     "react-native-svg": "9.4.0",
+    "react-native-syntax-highlighter": "2.0.1",
```

Comment:
Please add entry to https://github.com/mattermost/mattermost-mobile/blob/master/NOTICE.txt

Reply:
Done 👍